Automattic/sensei
# Add existing questions modal

### Changes proposed in this Pull Request

* It introduces the existing questions modal to the quiz block.
* This PR doesn't contain the "Add selected" feature.

### Testing instructions

* Create many questions.
* Go to the lesson editor.
* In the quiz block, click on "Add existing questions", and make sure the modal works properly.
  * Filter the content, select/unselect, clear, etc.


### Screenshot / Video

https://user-images.githubusercontent.com/876340/108911588-3110d480-7606-11eb-8041-644525b37198.mov

Small window or many questions:

https://user-images.githubusercontent.com/876340/108913939-7387e080-7609-11eb-9666-a24461e9a171.mov

Cropped text (with title):

<img width="784" alt="Screen Shot 2021-02-23 at 18 58 59" src="https://user-images.githubusercontent.com/876340/108914073-9f0acb00-7609-11eb-94c3-f5d9d91c6d2b.png">
